lib/src/cartesian/components/BarGraphLabels.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Text, type SkFont } from "@shopify/react-native-skia";
+import type { ChartBounds, PointsArray } from "lib/src/types";
+
+export type BarLabelConfig = {
+  position: "top" | "bottom" | "left" | "right";
+  font: SkFont | null;
+  color?: string;
```

Comment:
We could technically allow this to be the Skia `Color` type  to fully support more colors for users.